tests/test_lab_init.py

```diff
@@ -22,26 +18,54 @@ class TestLabInit:
     @patch("instructlab.config.init.Repo.clone_from")
     def test_init_noninteractive(self, mock_clone_from):
         runner = CliRunner()
-        with runner.isolated_filesystem():
+        with runner.isolated_filesystem() as td:
+            os.environ["HOME"] = td
+            # lazily import this module as a result of the following issue:
+            # https://github.com/instructlab/instructlab/issues/1582
+            # First Party
+            from instructlab import lab
+            from instructlab.configuration import DEFAULTS
```

Comment:
This does not work reliable. You cannot guarantee that no other test module has imported `instructlab.lab` before this test runs.

Reply:
Is this a blocking requested change or can we move it to an issue? There are other PRs that are blocked waiting on this one.

Reply:
Fixed this locally.